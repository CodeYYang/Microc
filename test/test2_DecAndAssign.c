int main(){
    int x=2;
    print("%d",x);
}