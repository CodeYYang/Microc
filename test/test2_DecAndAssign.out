24 19 0 5 25 13 0 2 12 13 11 22 21 1